scanner/src/main/kotlin/scanners/Askalono.kt

```diff
@@ -156,10 +156,19 @@ class Askalono(name: String, config: ScannerConfiguration) : LocalScanner(name,
 
     override fun generateSummary(startTime: Instant, endTime: Instant, path: File, result: JsonNode): ScanSummary {
         val licenseFindings = result.map {
-            // TODO: Set the location of the finding properly
+            val relativePath = if (path.isFile) {
+                if (path.isAbsolute) {
+                    path.relativeTo(File(".").absoluteFile)
+                } else {
+                    path
+                }
+            } else {
+                File(it["Path"].textValue()).relativeTo(path.absoluteFile)
```

Comment:
Why can't this be the default, does the result not contain a "Path" if the input is a single file? Please add code comments explaining this, I found it hard to understand the logic behind this.

Reply:
I've now introduced a helper function for that.